data/campaigns/Eastern_Invasion/scenarios/10_Dark_Sanctuary.cfg

```diff
@@ -0,0 +1,2776 @@
+        [/message]
+    [/event]
+
+    [event]
+        name=side 1 turn 2
+
+        [message]
+            speaker=Dacyn
+            message= _ "Ah, I remember this place from my studies. Now that I think of it, even when we were students, Ravan seemed very interested in the magic of darkness. Apparently, the balance between light and dark would be crucial to the safety of Irdya in the future, at least according to Ravan’s theory. In hindsight, that was all nonsense."
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "The irony seems lost on you."
+        [/message]
+        [message]
+            speaker=Dacyn
+            message= _ "Back then this place was full of traps and undead, and it will no doubt be worse now that the wild beasts have had a chance to move in."
+        [/message]
+
+        {MOVE_UNIT id=Dacyn 36 18}
+        {MODIFY_UNIT id=Dacyn facing nw}
+        [delay]
+            time=500
+        [/delay]
+
+        [message]
+            speaker=Dacyn
+            message= _ "Ravan sealed this place before our last meeting. I need a moment to remember the incantation."
+        [/message]
+        [delay]
+            time=1500
+        [/delay]
+        {DACYN_FAKEATTACK}
+        [message]
+            speaker=Dacyn
+            #po: the "red star" refers to the Ruby of Fire, and the "red stone" refers to the Amulet of the Veil FYI but these are not to be explicitly mentioned
+            #po: the "black pool" would be located near or under Weldyn, but again not to be explicitly mentioned
+            message= _ "The red star blazes,
+The red stone bleeds.
+The black pool swallows all voices,
+The darkness between worlds opens its maw."
+        [/message]
+        [delay]
+            time=500
+        [/delay]
+        [sound]
+            name=gate-fall.ogg
+        [/sound]
+
+        {MODIFY_TERRAIN Urc^Pr/ 31 17}
+        {MODIFY_TERRAIN Urc^Pr/ 32 16}
+        {MODIFY_TERRAIN Urc^Pr/ 33 16}
+        {MODIFY_TERRAIN Urc^Pr/ 34 15}
+        {MODIFY_TERRAIN Urc^Pr/o 34 18} {MOVE_UNIT (x,y=34,18) 34 19}
+        {MODIFY_TERRAIN Urc^Pr/o 35 18} {MOVE_UNIT (x,y=35,18) 35 19}
+        {MODIFY_TERRAIN Urc^Pr/o 36 17} {MOVE_UNIT (x,y=36,17) 36 18}
+        {MODIFY_TERRAIN Urc^Pr/o 37 17} {MOVE_UNIT (x,y=37,17) 37 18}
+        [redraw]
+            clear_shroud=yes
+            side=1
+        [/redraw]
+
+        [message]
+            speaker=Hahid al-Ali
+            message= _ "(shudders) Ugh, spellcasting always gives me the shivers. There’s a reason the more civilized people of my homeland don’t trifle with magic!"
+        [/message]
+        [message]
+            speaker=Terraent
+            message= _ "For a mage of the light, the words you speak are dark indeed."
```

Comment:
Earlier, Owaec addresses Dacyn as a 'mage of the Light'. Should 'light' similarly be capitalised here for consistency?